cyences_app_for_splunk/bin/upgrade_steps.py

```diff
@@ -109,6 +109,17 @@ def upgrade_5_0_0(session_key, logger):
     conf_manager.update_macro(SOC_EMAIL_CONFIG_MACRO, {"definition": default_emails})
     logger.info("Updated the {} macro with the default emails configured for the cyences_send_email_action.".format(SOC_EMAIL_CONFIG_MACRO))
 
+    vpn_login_attempt_old_macro_name = "cs_authentication_vpn_login_attemps_outside_working_hour_filter"
+    vpn_login_attempt_new_macro_name = "cs_authentication_vpn_login_attempts_outside_working_hour_filter"
+    try:
+        old_macro_definition = conf_manager.get_conf_stanza('macros', vpn_login_attempt_old_macro_name)[0]["content"]["definition"]
+        conf_manager.update_macro(vpn_login_attempt_new_macro_name, old_macro_definition)
```

Comment:
here we have to pass defination as a dict:

conf_manager.update_macro(vpn_login_attempt_new_macro_name, {"definition": old_macro_definition})

Reply:
done